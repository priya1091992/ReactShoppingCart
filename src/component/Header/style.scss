.header {
  background-color: lightGrey;
  padding: 10px;
}
