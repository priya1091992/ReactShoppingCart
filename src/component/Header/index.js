import React, { Component } from 'react';
import './style.scss';

class Header extends Component {
  render () {
    return (
      <div className="header">
        {`Header`}
      </div>
    )
  }
}

export default Header
