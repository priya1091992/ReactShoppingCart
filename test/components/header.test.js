import React from 'react';
import Header from './../../src/component/Header/index';
import renderer from 'react-test-renderer';

test('Footer component', () => {
  const component = renderer.create(
    <Header />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
