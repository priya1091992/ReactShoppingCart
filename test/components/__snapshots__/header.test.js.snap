// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component 1`] = `
<div
  className="header"
>
  Header
</div>
`;
